tests/ImageSharp.Tests/Formats/Tiff/TiffEncoderTests.cs

```diff
@@ -475,12 +476,20 @@ private static void TestStripLength<TPixel>(TestImageProvider<TPixel> provider,
                 }
             }
 
+            // TODO: Ask Brian about this. It seems like some of the images used
```

Comment:
The reason the image is different from the input image is because a palette is used. Due to the quantization step the output can differ slightly. I have checked the output and it looks visually identical. 

@JimBobSquarePants I will go ahead and change the particular test to use a tolerant comparer, if you dont mind.

Reply:
Yeah go ahead, thanks!